frontend/src/component/ai/AIChat.tsx

```diff
@@ -134,24 +142,32 @@ export const AIChat = () => {
 
     return (
         <StyledAIChatContainer>
-            <StyledChat>
-                <AIChatHeader
-                    onNew={() => setMessages(initialMessages)}
-                    onClose={() => setOpen(false)}
-                />
-                <StyledChatContent>
-                    <AIChatMessage from='assistant'>
-                        Hello, how can I assist you?
-                    </AIChatMessage>
-                    {messages.map(({ role, content }, index) => (
-                        <AIChatMessage key={index} from={role}>
-                            {content}
+            <StyledResizable
+                handlers={['top-left', 'top', 'left']}
+                minSize={{ width: '270px', height: '200px' }}
+                maxSize={{ width: '90vw', height: '90vh' }}
+                defaultSize={{ width: '320px', height: '450px' }}
+                onResize={scrollToEnd}
```

Comment:
I can see cases where this would be annoying, ie you scroll up in history and then resize to read something without too many line breaks. But not sure the need would arise in here

Reply:
If you try this locally I think you'll understand why I did this. If we don't do something like this, whenever you resize, the conversation content container will anchor to the top left.

Reply:
I'll keep it for now but we can remove this behavior if we get feedback about it.